hazelcast/src/test/java/com/hazelcast/test/TestHazelcastInstanceFactory.java

```diff
@@ -75,7 +75,16 @@ public HazelcastInstance newHazelcastInstance() {
     public HazelcastInstance newHazelcastInstance(Config config) {
         if (mockNetwork) {
             init(config);
-            NodeContext nodeContext = registry.createNodeContext(pickAddress());
+            NodeContext nodeContext = registry.createNodeContext(pickAddress(), false);
+            return HazelcastInstanceFactory.newHazelcastInstance(config, null, nodeContext);
+        }
+        return HazelcastInstanceFactory.newHazelcastInstance(config);
+    }
+
+    public HazelcastInstance newHazelcastInstance(Config config, boolean firewalling) {
```

Comment:
then I'd suggest to merge the 2 mock network managers into 1 and always use it. this will remove number of moving parts / configuration options from our test.